### DISCUSSION TITLE
plextraktsync broken for me (Need help)

### DISCUSSION THREAD

**Discussion by rawn7702** [Taxel/PlexTraktSync#1056] [Q&A] [answered]
I successfully setup plextraktsync a few months ago by following the guide. It synced properly for a long time until now, it could not sync my watch time when I executed `plextraktsync watch` 

So I decided to follow the guide again and redid the steps from the beginning. This time I get a string of errors most of which say `bad interpreter` and `no such file or directory`

I do not know what is going on. The trakt API is still active on my account.

I am on a Mac.

Can someone help?
Thank you.

**Comment by glensc** [comment]
massive lack of information, no steps to reproduce. no links to guides, no info on what you do what commands you ran. exact error messages and prior commands ran not disclosed.

I'm just guessing your files are DOS EOL somehow. figure out yourself how you transferred the files.

use supported installation methods:
- https://github.com/Taxel/PlexTraktSync#installation

not some vague "guides", which I repeat: you didn't even disclose!

**Reply by rawn7702** [reply]
> massive lack of information, no steps to reproduce. no links to guides, no info on what you do what commands you ran. exact error messages and prior commands ran not disclosed.
> 
> I'm just guessing your files are DOS EOL somehow. figure out yourself how you transferred the files.
> 
> use supported installation methods:
> 
> * https://github.com/Taxel/PlexTraktSync#installation
> 
> not some vague "guides", which I repeat: you didn't even disclose!

Hello, I am sorry for the trouble.
I was able to reinstall it and get the error again.
Here it is:
```
INFO     PlexTraktSync [0.21.4]                                                 
ERROR    Plex connection error:                                                 
         HTTPSConnectionPool(host='192-168-1-36.aa6213eecf094ff4b57857ebeace3e21
         .plex.direct', port=32400): Max retries exceeded with url: / (Caused by
         ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at      
         0x1049ddcc0>, 'Connection to                                           
         192-168-1-36.aa6213eecf094ff4b57857ebeace3e21.plex.direct timed out.   
         (connect timeout=30)')), local url http://localhost:32400 didn't       
         respond either. Check PLEX_LOCALURL in .env file.                      
Plex connection error: HTTPSConnectionPool(host='192-168-1-36.aa6213eecf094ff4b57857ebeace3e21.plex.direct', port=32400): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x1049ddcc0>, 'Connection to 192-168-1-36.aa6213eecf094ff4b57857ebeace3e21.plex.direct timed out. (connect timeout=30)')), local url http://localhost:32400 didn't respond either. Check PLEX_LOCALURL in .env file.
```

**Reply by glensc** [reply]
different problem, different discussion. 

also you should write what was your problem and h0w you solved so other users finding your report are able to troubleshoot themselves.

**Reply by rawn7702** [reply]
I apologise for my mistake. Do I edit my original post or create a new one?

**Reply by glensc** [reply]
here you write your solution to the problem you had, and new problem you create new topic.

**Reply by rawn7702** [reply, marked as answer]
~SOLUTION (partial): deleted the source folder and reinstalled plextraktsync again to regenerate the error.~

**Reply by glensc** [reply]
not useful for other users, as they don't have info how you installed in first place and how you installed now.

also your guides you referred are still uknown.
